### PR TITLE
Mappers

### PR DESCRIPTION
**Pull request by BQuinteros**
In this Pull Request, I added mappers to transform de response data to data of entities I created. 
Image

### CODE REVIEW

**Comment by BQuinteros** [comment]
Added country mapper in the last commit

**Review by SartiMau — APPROVED** [review]

**Review by hernanhorquin96 — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

File: WheaterApp/data/src/main/java/com/example/data/services/WeatherCityServiceImpl.kt [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
+    private val mapper: WeatherCityMapperService = WeatherCityMapperService()
 
-    override fun getCityById(id: Int): Boolean {
+    override fun getCityById(id: Int): Result<City> {
         val queryHashMap = LinkedHashMap<String, String>()
         queryHashMap.put(ID, id.toString())
         queryHashMap.put(APPID, WeatherRequestGenerator.API_KEY)
         val callResponse = api.createService(WeatherApi::class.java).getCityById(queryHashMap)
         val response = callResponse.execute()
         if (response != null) {
```
Comment (lucianoeli) [inline WheaterApp/data/src/main/java/com/example/data/services/WeatherCityServiceImpl.kt]: use let?. here

**Review by lucianoeli — APPROVED** [review]

File: WheaterApp/data/src/main/java/com/example/data/mappers/WeatherCityMapperService.kt [hunk 2]
```diff
@@ -24,9 +24,9 @@ class WeatherCityMapperService : BaseMapperRepository<CityResponse, City> {
     private fun transformMain(type: MainResponse): Main {
         type.apply {
             return Main(
-                    temp,
-                    temp_min,
-                    temp_max
+                    temp- DEGREE_KELVIN,
```
Comment (SartiMau) [inline WheaterApp/data/src/main/java/com/example/data/mappers/WeatherCityMapperService.kt]: is that white space working?

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: WheaterApp/data/src/main/java/com/example/data/services/WeatherCityServiceImpl.kt (hunk 1 above)
Comment (BQuinteros) [reply WheaterApp/data/src/main/java/com/example/data/services/WeatherCityServiceImpl.kt]: Done.

File: WheaterApp/data/src/main/java/com/example/data/mappers/WeatherCityMapperService.kt (hunk 2 above)
Comment (BQuinteros) [reply WheaterApp/data/src/main/java/com/example/data/mappers/WeatherCityMapperService.kt]: Done

**Comment by BQuinteros** [comment]
image

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**